src/aws/k8s/k8s_module/main.tf

```diff
@@ -164,20 +182,24 @@ resource "aws_spot_instance_request" "node" {
 
   provisioner "remote-exec" {
     inline = [
-      "sudo echo 'export CONTROL_PLANE_IP=${aws_instance.control-plane.private_ip}' >> /src/scripts/vars.sh",
-      "sudo echo 'kubeadm join --token=${local.bootstraptoken} --discovery-token-unsafe-skip-ca-verification ${aws_instance.control-plane.private_ip}:6443' >> /src/output/.kubeadmin_init",
+      "sudo echo 'export CONTROL_PLANE_IP=${aws_eip.control-plane.private_ip}' >> /src/scripts/vars.sh",
```

Comment:
asta o rulezi si in remote exec-ul de mai sus, nu e redundanta?

Reply:
nu e, una se ruleaza in control-plane, alta in node.